docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+          { id: "end_time", header: "End Time", sortable: true },
+          { id: "run_creation_time", header: "Creation Time", sortable: true },
+          { id: "status", header: "Status" },
+        ],
+        links: [
+          {
+            displayName: "Start Server Experiment",
+            path: "/server-experimentation/start",
+          },
+        ],
+      },
+    },
+    viewExperimentRun: {},
+  },
+  "@clutch-sh/server-experimentation": {
+    startExperiment: {
+      componentProps: {
+        upstreamClusterTypeSelectionEnabled: true,
+      },
+      hideNav: true,
+    },
+  },
+```
+
+#### Backend Server
+
+Below configuration will spin up all the required modules and services to store the data coming from frontend into Postgres database.
+
+```yaml title="backend/clutch-config.yaml"
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.api
+  - name: clutch.module.chaos.lyftexperimentation
+  - name: clutch.module.chaos.serverexperimentation
+services:
+  ...
+  - name: clutch.service.db.postgres
+      typed_config:
+        "@type": types.google.com/clutch.config.service.db.postgres.v1.Config
+        connection:
+          host: ${RDS_HOST}
+          port: ${RDS_PORT}
+          user: ${RDS_USER}
+          ssl_mode: REQUIRE
+          dbname: ${RDS_NAME}
+          password: ${CREDENTIALS_CLUTCH_TO_RDS_PASSWORD}
+  - name: clutch.service.chaos.experimentation.store
+```
+
+#### xDS Management Server 
+
+Below is the configuration for spinning up xDS server. For details about the fields, take a look at the [xds config proto](https://github.com/lyft/clutch/blob/main/api/config/module/chaos/experimentation/xds/v1/xds.proto).
+
+```yaml title="backend/clutch-xds-config.yaml"
+...
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.xds
+    typed_config:
+      "@type": types.google.com/clutch.config.module.chaos.experimentation.xds.v1.Config
+      rtds_layer_name: <RTDS_LAYER_NAME>                     // "rtds_layer"
+      cache_refresh_interval: <CACHE_REFRESH_INTERNAL>       // "5s"
+      ingress_fault_runtime_prefix: <INGRESS_FAULT_PREFIX>   // "fault.http"
+      egress_fault_runtime_prefix: <EGRESS_FAULT_PREFIX>     // "fault.http.egress"
+      resource_ttl: <RESOURCE_TTL>                           // "20s"
+      heartbeat_interval: <HEARTBEAT_INTERVAL>               // "5s"
+      ecds_allow_list: <LIST_OF_ECDS_ENALBED_CLUSTERS>       // ["foo", "bar"]
+services:
+  - name: clutch.service.db.postgres
+    typed_config:
+      "@type": types.google.com/clutch.config.service.db.postgres.v1.Config
+      connection:
+        host: ${RDS_HOST}
+        port: ${RDS_PORT}
+        user: ${RDS_USER}
+        ssl_mode: REQUIRE
+        dbname: ${RDS_NAME}
+        password: ${CREDENTIALS_CLUTCH_TO_RDS_PASSWORD}
+  - name: clutch.service.chaos.experimentation.store
+```
+
+Keep in mind that both backend config and xDS config needs to connect to the same Postgres database.
```

Comment:
```suggestion
Keep in mind that both backend config and xDS config need to connect to the same Postgres database.
```

Reply:
I thought that it was not possible to have 2 separate database configured with postgres service as of now?

Reply:
Since both configs (clutch and xds) need Postgres, I think its better to clarify.